linera-core/src/chain_worker/state/attempted_changes.rs

```diff
@@ -496,7 +496,7 @@ where
         &mut self,
         latest_heights: Vec<(Target, BlockHeight)>,
     ) -> Result<(), WorkerError> {
-        let mut height_with_fully_delivered_messages = BlockHeight::ZERO;
+        let mut height_with_fully_delivered_messages = None;
```

Comment:
(Alternatively: `next_delivery_height: BlockHeight::ZERO`, and add `1` below.)

Reply:
But we would to subtract 1 as well.